packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -230,12 +251,15 @@ export default class TableResize implements EditorPlugin {
         if (this.insertingState == ResizeState.None) {
             return;
         }
+
+        this.isRTL = getComputedStyle(this.editor.getDocument().body, 'direction') == 'rtl';
```

Comment:
this.isRTL is used before this line. Need to do this earlier. And no need to calculate everytime when mouse move.